swingset/src/main/java/com/nqadmin/swingset/SSBaseComboBox.java

```diff
@@ -189,6 +221,12 @@ protected BaseGlazedModel() {
 		}
 	}
 
+	/**
+	 * The subclass implements this to provide the logger.
+	 * @return logger
+	 */
+	protected abstract Logger logger();
+
```

Comment:
So this will use the logger from the subclass? The way I have the logging configured it shows the calling class so I generally prefer to let each class have it's own logger. That way we can tell if we're looking at a call from Base vs Combo vs DBCombo. My preference would be to give SSBaseComboBox it's own logger.

Reply:
OK